src/Console/BackupCommand.php

```diff
@@ -86,8 +86,14 @@ protected function saveSettings($indexName)
 
     protected function saveSynonyms($indexName)
     {
+<<<<<<< HEAD
```

Comment:
This looks like an accidental committed line?

Reply:
Oops ^^